### ISSUE TITLE
[STORM-945] <DefaultRolloverStrategy> element is not a policy,and should not be putted in the <Policies> element.

### ISSUE THREAD

**Issue by jira-importer** [apache/storm#4783]
<div class="code panel" style="border-width: 1px;"><div class="codeContent panelContent">
<pre class="code-java">➜  storm-current  bin/storm nimbus
Running: /usr/share/jdk-current/bin/java -server -Ddaemon.name=nimbus ... 
...
2015-07-17 10:23:36,776 ERROR Policies has no parameter that matches element DefaultRolloverStrategy
2015-07-17 10:23:36,787 ERROR Policies has no parameter that matches element DefaultRolloverStrategy
2015-07-17 10:23:36,795 ERROR Policies has no parameter that matches element DefaultRolloverStrategy
</pre>
</div></div>

---
<details><summary><i>Originally reported by <a title="caofangkun" href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=caofangkun">caofangkun</a>, imported from: <a href="https://issues.apache.org/jira/browse/STORM-945" target="_blank"><DefaultRolloverStrategy> element is not a policy,and should not be putted in the <Policies> element.</a></i></summary>
<i><ul>
<li><b>assignee</b>: <a title="caofangkun" href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=caofangkun">caofangkun</a>
<li><b>status</b>: Resolved
<li><b>priority</b>: Minor
<li><b>resolution</b>: Fixed
<li><b>resolved</b>: 2015-07-30T15:51:33+01:00
<li><b>imported</b>: 2025-01-24
</ul></i>
</details>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>GitHub user caofangkun opened a pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/638" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/638</a></p>

<p>    <a href="https://issues.apache.org/jira/browse/STORM-945" title="<DefaultRolloverStrategy> element is not a policy,and should not be putted in the <Policies> element." class="issue-link" data-issue-key="STORM-945"><del>STORM-945</del></a>:<DefaultRolloverStrategy> element is not a policy,and should not be putted in the <Policies> element</p>

<p>    ```<br/>
    -> storm-current  bin/storm nimbus<br/>
    Running: /usr/share/jdk-current/bin/java -server -Ddaemon.name=nimbus ... ...<br/>
    2015-07-17 10:23:36,776 ERROR Policies has no parameter that matches element DefaultRolloverStrategy<br/>
    2015-07-17 10:23:36,787 ERROR Policies has no parameter that matches element DefaultRolloverStrategy<br/>
    2015-07-17 10:23:36,795 ERROR Policies has no parameter that matches element DefaultRolloverStrategy<br/>
    ```</p>

<p>You can merge this pull request into a Git repository by running:</p>

<p>    $ git pull <a href="https://github.com/caofangkun/apache-storm" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/caofangkun/apache-storm</a> storm-945</p>

<p>Alternatively you can review and apply these changes as the patch at:</p>

<p>    <a href="https://github.com/apache/storm/pull/638.patch" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/638.patch</a></p>

<p>To close this pull request, make a commit to your master/trunk branch<br/>
with (at least) the following in the commit message:</p>

<p>    This closes #638</p>

<hr />
<p>commit 740f02ee0025cc0307a1422205f9612ce18d3652<br/>
Author: caofangkun <caofangkun@gmail.com><br/>
Date:   2015-07-17T03:35:11Z</p>

<p>    <a href="https://issues.apache.org/jira/browse/STORM-945" title="<DefaultRolloverStrategy> element is not a policy,and should not be putted in the <Policies> element." class="issue-link" data-issue-key="STORM-945"><del>STORM-945</del></a>:<DefaultRolloverStrategy> element is not a policy,and should not be putted in the <Policies> element</p>

<hr />

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user HeartSaVioR commented on the pull request:</p>

<p>    <a href="https://github.com/apache/storm/pull/638#issuecomment-126351254" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/638#issuecomment-126351254</a></p>

<p>    +1. I confirmed that works with 0.11.x-SNAPSHOT. <br/>
    Since it is misconfiguration of log4j2 and it's introduced to 0.10.0-beta, we'd be better to apply it to 0.10.x-branch, too.</p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=githubbot">githubbot</a>:</i>
<p>Github user asfgit closed the pull request at:</p>

<p>    <a href="https://github.com/apache/storm/pull/638" class="external-link" target="_blank" rel="nofollow noopener">https://github.com/apache/storm/pull/638</a></p>

**Comment by jira-importer** [comment]
<i><a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=kabhwan">kabhwan</a>:</i>
<p>Thanks <a href="https://issues.apache.org/jira/secure/ViewProfile.jspa?name=caofangkun" class="user-hover" rel="caofangkun">caofangkun</a><br/>
I merged it to master and 0.10.x-branch.</p>

**Comment by jira-importer** [comment]
<i>[Duplicates: <a href="https://github.com/apache/storm/issues?q=in%3Atitle%20STORM-925">STORM-925</a>]</i>
